backend/app/util/session_id.py

```
@@ -24,27 +25,36 @@ async def session_id_management(email: str) -> str:
         db: Database session
         email: The email of the user.
         session_id: The session ID.
     """
-    async with get_async_db_session() as db:
-        await session_id_crud.save_session_id(db=db, email=email, session_id=session_id)
+    try:
+        async with get_async_db_session() as db:
+            await session_id_crud.save_session_id(
+                db=db, email=email, session_id=session_id
+            )
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)